Script/TypeScript/AtomicExtended.d.ts

```diff
@@ -0,0 +1,115 @@
+// Manual typescript definitions from various CPP and JS files that can't yet be autogenerated
+// no-default-lib and modified lib.d.ts used to allow redefinition of self
+/// <reference no-default-lib="true"/>
+/// <reference path="./lib.d.ts" />
+/// <reference path="./Atomic.d.ts" />
+
+// From JSAtomic.cpp
+declare function print(val: any);
+declare function assert();
+declare function js_module_read_file(path: string);
+declare function openConsoleWindow();
+declare function getVM(): Atomic.JSVM;
+declare function getEngine(): Atomic.Engine;
+declare function getGraphics(): Atomic.Graphics;
+declare function getRenderer(): Atomic.Renderer;
+declare function getResourceCache(): Atomic.ResourceCache;
+declare function getInput(): Atomic.Input;
+declare function getFileSystem(): Atomic.FileSystem;
+declare function getNetwork(): Atomic.Network;
+declare function script(script: string): boolean;
+declare function destroy(node: Atomic.Node): boolean;
+declare function destroy(scene: Atomic.Scene): boolean;
+declare function destroy(component: Atomic.JSComponent): boolean;
+
```

Comment:
There are both get accessors and properties for subsystems (eg Atomic.getGraphics and Atomic.graphics). I see the accessors haven't been added to AtomicWork.d.ts, is this intentional or can I add them.